utility/test/module_test.go

```diff
@@ -8,6 +8,7 @@ import (
 	"os"
 	"testing"
 
+	typesPers "github.com/pokt-network/pocket/persistence/types"
```

Comment:
`Utility mod` added import from `Persistence mod`. 

We're trying to get rid of this cross module sharing

Reply:
Perhaps we could move module creation for testing purposes under test_artifacts to hide away the implementation details of the other modules when we are testing one

Reply:
Created https://github.com/pokt-network/pocket/issues/239

Reply:
@Olshansk that is likely going to be a somewhat sophisticated AST rule but should be doable! 👍

Reply:
I'm not sure how this was resolved. We just added an issue to automatically check